docs/spfx/viva/get-started/actions/geolocation/GeolocationDocumentation.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Documentation for Geolocation action in SharePoint Adaptive Card Extension
+description: Geolocation is a new action that the SharePoint Adaptive Card Extension framework supports, which enables third party developers to come up with their location specific scenarios.
+ms.date: 03/04/2022
+ms.prod: sharepoint
+ms.localizationpriority: high
+---
+# Location capabilities in Adaptive Card Extension
+
+> [!NOTE]
+> This tutorial assumes that you have installed the SPFx v1.14 and have already built your first SharePoint Adaptive Card Extension.<br/>
```

Comment:
Location APIs are not in 1.14 GA, right? I guess we should change this by saying "1.15 beta"?

Reply:
yes please, as when developers install 1.14 they will not find those APIs.

Reply:
@lucabandMSFT, I have made the changes. Please have a look at the new commit.
Closing this comment for now.